smithy4s-client/src/main/smithy/kinesis.2023-02-01.json

```diff
@@ -0,0 +1,6884 @@
+            "traits": {
+              "smithy.api#documentation": "<p>Specify this input parameter to distinguish data streams that have the same name. For\n            example, if you create a data stream and then delete it, and you later create another\n            data stream with the same name, you can use this input parameter to specify which of the\n            two streams you want to list the consumers for. </p>\n         <p>You can't specify this parameter if you specify the NextToken parameter. </p>"
+            }
+          }
+        },
+        "traits": {
+          "smithy.api#input": {}
+        }
+      },
+      "com.amazonaws.kinesis#ListStreamConsumersInputLimit": {
+        "type": "integer",
+        "traits": {
+          "smithy.api#range": {
+            "min": 1,
+            "max": 10000
+          }
+        }
+      },
+      "com.amazonaws.kinesis#ListStreamConsumersOutput": {
+        "type": "structure",
+        "members": {
+          "Consumers": {
+            "target": "com.amazonaws.kinesis#ConsumerList",
+            "traits": {
+              "smithy.api#documentation": "<p>An array of JSON objects. Each object represents one registered consumer.</p>"
+            }
+          },
+          "NextToken": {
+            "target": "com.amazonaws.kinesis#NextToken",
+            "traits": {
+              "smithy.api#documentation": "<p>When the number of consumers that are registered with the data stream is greater than\n            the default value for the <code>MaxResults</code> parameter, or if you explicitly\n            specify a value for <code>MaxResults</code> that is less than the number of registered\n            consumers, the response includes a pagination token named <code>NextToken</code>. You\n            can specify this <code>NextToken</code> value in a subsequent call to\n                <code>ListStreamConsumers</code> to list the next set of registered consumers. For\n            more information about the use of this pagination token when calling the\n                <code>ListStreamConsumers</code> operation, see <a>ListStreamConsumersInput$NextToken</a>.</p>\n         <important>\n            <p>Tokens expire after 300 seconds. When you obtain a value for\n                    <code>NextToken</code> in the response to a call to\n                    <code>ListStreamConsumers</code>, you have 300 seconds to use that value. If you\n                specify an expired token in a call to <code>ListStreamConsumers</code>, you get\n                    <code>ExpiredNextTokenException</code>.</p>\n         </important>"
+            }
+          }
+        },
+        "traits": {
+          "smithy.api#output": {}
+        }
+      },
+      "com.amazonaws.kinesis#ListStreams": {
+        "type": "operation",
+        "input": {
+          "target": "com.amazonaws.kinesis#ListStreamsInput"
+        },
+        "output": {
+          "target": "com.amazonaws.kinesis#ListStreamsOutput"
+       
```

Comment:
TODO: This is supposed to be PositiveIntegerOpbect, but it is a workaround with a min of 0 due to a bug in kinesis-mock:

https://github.com/etspaceman/kinesis-mock/pull/418/files